test/core/create-from-json.test.ts

```diff
@@ -26,6 +26,14 @@ describe('fromJSON simple', () => {
       { name: 'e', version: '5.0.0' },
       { name: 'root', version: '0.0.0' },
     ].sort(helpers.depSort));
+    expect(graph.getDepPkgs().sort(helpers.depSort)).toEqual([
```

Comment:
Can you make this a separate test, and change the test name to explain what its actually testing? i.e.

```
describe('getPkgs', () => {
  it('does not return the root node', () => {
    // expect...
  })
})'
```

Also doesn't look like this test needs to be async.

Reply:
done. and removed the `async` keyword from all tests in this file

Reply:
💜